### ISSUE TITLE
strange output using beachballs

### ISSUE THREAD

**Issue by Thomas-Ulrich** [obspy/obspy#2114]
Hi,
I m experiencing a strange behaviour of Obspy when plotting beachballs.
Try the following example:
mt0 gives me the expected beach ball associated with strike/dip/rake = 236/68/177.
mt1 is identical with mt0 but the decimals are latter troncated and gives a blue beach ball (?!?).
Any idea why?

Thanks in advance,

Thomas

```python
from obspy.imaging.mopad_wrapper import beachball
mt0 = [6.07624723e+19,         -6.34127963e+19,        2.65032397e+18,         2.65142658e+19,         -1.75252565e+19,         -2.10743112e+19]
mt1 = [6.0762472348250243e+19, -6.341279632309221e+19, 2.6503239748419671e+18, 2.6514265814783787e+19, -1.7525256484369068e+19, -2.1074311214873211e+19]
beachball(mt0, size=40, linewidth=2, facecolor='b',mopad_basis = 'NED', xy=(-70, 80))
beachball(mt1, size=40, linewidth=2, facecolor='b',mopad_basis = 'NED', xy=(-70, 80))
```

my obspy: 1.1.0-py27_1 conda-forge

**Comment by krischer** [comment]
I can reproduce this. We've had some numerical stability issues with very large numbers before - maybe we should start to internally normalize moment tensor before we plot them?

You can work-around this particular issue by normalizing the tensor components. This should result in more stable plots.

```
mt1 = np.array(mt1)
mt1 /= np.linalg.norm(mt1)
```

Then the plots work for both cases.

**Comment by Thomas-Ulrich** [comment]
Ok, 

Thank-you Lion for the support and for this quick workaround.

Thomas.

**Comment by megies** [comment]
> I can reproduce this. We've had some numerical stability issues with very large numbers before - maybe we should start to internally normalize moment tensor before we plot them?

sounds like a simple workaround and easy fix  :+1:

**Comment by megies** [comment]
closed by #2125
